File: Beetle.NetPackage/Samples/c#/WP8/BeetleNPDemo/Beetle.NetPackage/IMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
/**
 * Copyright © henryfan 2013
 * Created by henryfan on 13-7-30.
 * homepage:www.ikende.com
 * email:henryfan@msn.com
 */
namespace Beetle.NetPackage
{
    public interface IMessage
    {
        void Load(IDataReader reader);

        void Save(IDataWriter writer);
    }
}

File: BeetleDemo/Examples/Android/BeetleNPDemo/src/beetle/netpackage/IMessage.java
package beetle.netpackage;
import java.io.DataInputStream;
import java.io.DataOutputStream;

/**
 * Copyright © henryfan 2013
 * Created by henryfan on 13-7-30.
 * homepage:www.ikende.com
 * email:henryfan@msn.com
 */
public interface IMessage {
    void Load(DataInputStream stream) throws Exception;
    void Save(DataOutputStream stream) throws Exception;
}
